openfecli/commands/gather.py

```diff
@@ -171,24 +180,35 @@ def _get_ddgs(legs, error_on_missing=True):
 def _write_ddg(legs, writer, allow_partial):
     DDGs = _get_ddgs(legs, error_on_missing=not allow_partial)
     writer.writerow(["ligand_i", "ligand_j", "DDG(i->j) (kcal/mol)",
-                      "uncertainty (kcal/mol)"])
+                     "uncertainty (kcal/mol)"])
     for ligA, ligB, DDGbind, bind_unc, DDGhyd, hyd_unc in DDGs:
-        name = f"{ligB}, {ligA}"
         if DDGbind is not None:
-            DDGbind, bind_unc = format_estimate_uncertainty(DDGbind,
-                                                            bind_unc)
+            DDGbind, bind_unc = format_estimate_uncertainty(DDGbind, bind_unc)
             writer.writerow([ligA, ligB, DDGbind, bind_unc])
         if DDGhyd is not None:
-            DDGhyd, hyd_unc = format_estimate_uncertainty(DDGbind,
-                                                          bind_unc)
+            DDGhyd, hyd_unc = format_estimate_uncertainty(DDGhyd, hyd_unc)
             writer.writerow([ligA, ligB, DDGhyd, hyd_unc])
 
 
-def _write_dg_raw(legs, writer, allow_partial):
+def _write_raw(legs, writer, allow_partial=True):
+    writer.writerow(["leg", "ligand_i", "ligand_j", "DG(i->j) (kcal/mol)",
+                     "uncertainty (kcal/mol)"])
```

Comment:
```suggestion
                     "MBAR uncertainty (kcal/mol)"])
```
sorry just a slight precision